libs/protocol-ui/src/components/molecules/CreatableSelect/CreatableSelect.tsx

```diff
@@ -6,6 +6,16 @@ import FormLabel from '../../atoms/FormLabel';
 import HelperText from '../../atoms/HelperText';
 import ErrorMessage from '../../atoms/ErrorMessage';
 import { UseFormReturn } from 'react-hook-form/dist/types/form';
+import {components} from 'react-select';
+
+export function SelectContainer(props: any) {
+  return (
+     <components.SelectContainer
+        {...props}
+        innerProps={Object.assign({}, props.innerProps, { 'data-cy': 'daoCreatableSelect-testing' })}
```

Comment:
We'll want to make this passed in via props optionally instead of hardcoded -- this'll result in every CreatableSelect having that data-id